_includes/head.html

```diff
@@ -9,7 +9,7 @@
   <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
   <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#456BD9">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700" crossorigin>
-  <link rel="stylesheet" href="/main.css">
+  <link rel="stylesheet" type="text/css" href="https://cloudfour-patterns.netlify.com/assets/toolkit/styles/toolkit.css">
```

Comment:
- `type="text/css"` is not necessary.
- I believe there are reasons we decided against hot-linking our pattern library CSS, but I can't remember what they were.
- If we are going to load the CSS from a separate place, we should prefer the `.app` version of the Netlify domain to avoid a redirect.

Reply:
purgecss is revived